proto/axelar/vote/exported/v1beta1/types.proto

```diff
@@ -59,3 +59,15 @@ enum PollState {
   POLL_STATE_COMPLETED = 2 [ (gogoproto.enumvalue_customname) = "Completed" ];
   POLL_STATE_FAILED = 3 [ (gogoproto.enumvalue_customname) = "Failed" ];
 }
+
+// PollParticipants should be embedded in poll events in other modules
+message PollParticipants {
+  uint64 poll_id = 1 [
+    (gogoproto.customname) = "PollID",
+    (gogoproto.customtype) = "PollID",
+    (gogoproto.nullable) = false
+  ];
+  repeated bytes participants = 2
+      [ (gogoproto.casttype) =
+            "github.com/cosmos/cosmos-sdk/types.ValAddress" ];
```

Comment:
```suggestion
  repeated bytes participants = 2
      [ (gogoproto.nullable) = false,
        (gogoproto.casttype) =
            "github.com/cosmos/cosmos-sdk/types.ValAddress" ];
```

Reply:
that doesn't do anything, slices are always references (nullable = false will pick a value type otherwise), so the default is always nil, no way around it